sdk/eventgrid/azure-eventgrid/azure/eventgrid/_consumer.py

```diff
@@ -34,11 +37,8 @@ def deserialize_cloud_events(self, cloud_event, **kwargs): # pylint: disable=no-
         try:
             cloud_event = CloudEvent._from_json(cloud_event, encode) # pylint: disable=protected-access
```

Comment:
this will need updating I believe once you change the [`_from_json` to `_to_json`](https://github.com/Azure/azure-sdk-for-python/blob/1b10b4e4b6252a856074de644602daea3b08866b/sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py#L36) :)

Reply:
see #16352

Reply:
so I just realize that currently the `_from_json` accepts a json string and returns a json/dict object?

and if I'm understanding correctly, we're gonna have a separate PR which have the method matching the functionality?
(probably we could be a bit verbose on the method name, like `_from_json_str`, `_to_json_object`-ish)